presto-product-tests/libs/download.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e;
+
+HIVE_JDBC_DOWNLOAD_URL='http://bdch-ftp.td.teradata.com:8081/nexus/service/local/artifact/maven/redirect?r=test-framework&g=org.apache.hive&a=hive-jdbc-all&v=1.0-SNAPSHOT&e=jar'
```

Comment:
Sure - this has to be addressed.
https://github.com/prestodb/presto-hive-apache/pull/17 is an approach to that :)